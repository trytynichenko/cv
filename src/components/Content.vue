<template>
  <div class="content">
    <p v-for="(item, i) in data" :key="i">
      {{ item }}
    </p>
    <p v-for="(item, i) in html" :key="i" v-html="item">
      {{ item }}
    </p>
  </div>
</template>

<script>
export default {
  name: 'Content',
  props: {
    data: Array,
    html: Array,
  },
};
</script>

<style scoped lang="scss">
  @import '../assets/styles/components/content';
</style>
