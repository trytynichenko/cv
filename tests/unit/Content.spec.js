import { shallow } from '@vue/test-utils';
import Content from '@/components/Content.vue';

describe('Content.vue', () => {
  const props = {
    html: [
      '<strong>html string</strong>',
    ],
  };

  const wrapper = shallow(Content, {
    propsData: props,
  });

  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
