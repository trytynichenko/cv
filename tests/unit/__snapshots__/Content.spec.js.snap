// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content.vue matches snapshot 1`] = `
<div class="content">
  <p><strong>html string</strong></p>
</div>
`;
